Closes: Temperate-Designs/sunflower-dash#9

AnirbanBall opened the pull request:
Added issue template to repository for bug reports #9

AnirbanBall commented:
I made it in bug_report.yml instead bug_report.md because nowadays yml is more  popular and github supports more of yml files issue templates .So thats why

nicolasbock commented on .github/ISSUE_TEMPLATE/bug_report.yaml:
```diff
@@ -0,0 +1,41 @@
+body:
+  - type: markdown
+    attributes:
+      value: |
```
I `placeholder` might work better here

nicolasbock commented on .github/ISSUE_TEMPLATE/bug_report.yaml:
```diff
@@ -0,0 +1,41 @@
+    attributes:
+      value: |
+        Thanks for taking the time to fill out our bug report form 🙏
+  - type: textarea
```
Please use `markdown` as this would allow to format the text here.

nicolasbock commented on .github/ISSUE_TEMPLATE/bug_report.yaml:
```diff
@@ -0,0 +1,41 @@
+      label: "👟 Reproduction steps"
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: "When I ..."
+  - type: textarea
```
Same here, `markdown`.

nicolasbock commented on .github/ISSUE_TEMPLATE/bug_report.yaml:
```diff
@@ -0,0 +1,41 @@
+      label: "👍 Expected behavior"
+      description: "What did you think would happen?"
+      placeholder: "It should ..."
+  - type: textarea
```
Same here, `markdown`.

nicolasbock reviewed (changes requested):
That's awesome! Thanks!

Please have a look at my suggestions.

nicolasbock reviewed (changes requested):
Please make this a separate PR.

AnirbanBall commented:
sure

AnirbanBall commented:
<img width="770" alt="myissue" src="https://user-images.githubusercontent.com/72156677/136027409-62231863-5d96-48a1-84af-c3fd80ca49b2.png">

@nicolasbock I have implemented this code  now in my repository it works fine and also the markdown too works .You can check the screenshot

you can even visit my repo
 https://github.com/AnirbanBall/project/issues/new/choose  this code I implemented on my project too 
u can check here and see what all changes should be made

nicolasbock reviewed (approved):
Ah, I didn't realize how `textarea` would look. Thanks so much!

AnirbanBall commented:
okey Thank you so much
